migrations_scripts/1_deploy_registry_and_add_adapters.js

```diff
@@ -155,6 +155,12 @@ const saiPoolAddress = '0xb7896fce748396EcFC240F5a0d3Cc92ca42D7d84';
 const daiPoolAddress = '0x29fe7D60DdF151E5b52e5FAB4f1325da6b2bD958';
 const usdcPoolAddress = '0x0034Ea9808E620A0EF79261c51AF20614B742B24';
 
+const yfiAddress = '0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e';
+const yearnRewardsStakingCurvePoolAddress = '0x0001FB050Fe7312791bF6475b96569D83F695C9f';
+const yearnRewardsStakingBalancerPoolAddress = '0x033E52f513F9B98e129381c6708F9faA2DEE5db5';
+const yearnRewardsStakingGovernancePoolAddress = '0x3A22dF48d84957F907e67F4313E3D43179040d6E';
+const yearnRewardsContractAddress = '0xcc9EFea3ac5Df6AD6A656235Ef955fBfEF65B862';
```

Comment:
fix to comply with contracts

YFI, CURVE_Y, BALANCER_DAI_YFI_98_2, BALANCER_YFI_CURVE_Y_2_98

Reply:
fixed but unsure of if I should capitalize the first word to reflect the contract declaration or lowercase to reflect the other addresses in the migrations file

Reply:
lowercase is ok, I changed names a bit in order not to use underscores in names)